client/common/directives/fileChooser/fileChooser.directive.coffee

```diff
@@ -0,0 +1,21 @@
+fileChooser = ->
+    restrict: 'AE'
+    replace: true
+    templateUrl: 'client/common/directives/fileChooser/fileChooser.directive.html'
+    scope:
+        file: '='
+    link: (scope, element, attrs) ->
+        scope.changeFile = (event) ->
+            firstFile = _.first(event.target.files)
+            if firstFile? && firstFile.name?
+                scope.file = firstFile
+                scope.$apply()
```

Comment:
Apply robi digest loopa we wszystkich scope'ach, od roota zaczynając. Digest tylko w lokalnym scopie